controllers/generators/cert.go

```diff
@@ -1,94 +1,94 @@
 package generators
 
-import (
-	"crypto/rand"
-	"crypto/rsa"
-	"crypto/x509"
-	"crypto/x509/pkix"
-	"encoding/pem"
-	"math/big"
-	"net"
-	"time"
-)
+// import (
```

Comment:
I think this may just be a lack of familiarity with the code on my part, but would it make sense to just delete this file?